docs/InjiWalletComponents.md

```diff
@@ -0,0 +1,53 @@
+# **Inji Wallet Components**
+
+<!-- TOC -->
+
+- [**Inji Wallet Components**](#inji-wallet-components)
+_ [**Inji Wallet**](#inji-wallet)
+_ [**Mimoto**](#mimoto)
+_ [**Inji Verify**](#inji-verify)
+_ [**Inji Certify**](#inji-certify)
+_ [**Data Share**](#data-share)
+_ [**ESignet**](#esignet) \* [\*\*Native Libraries](#native-libraries)
+<!-- TOC -->
+
+### **Inji Wallet**
+
+- The Inji Wallet is a mobile application for Android and iOS developed in react native.
+- It offers a secure, trustworthy, and dependable mobile Verifiable Credentials wallet designed to fulfil the following functions
+  - Download and store Verifiable Credentials
+  - Conduct offline face verification
```

Comment:
this sounds like a standalone feature, but I think this should be worded more to tell unlock the usage etc..